### PR TITLE
Subscriptions + webhooks support

### PR DESCRIPTION
**Pull request by jacoor**
- Also bumped version to 0.2
- major cleanup with model renames, etc.

### CODE REVIEW

**Comment by jacoor** [comment]
For the coverage complaining: https://coveralls.io/builds/11894465/source?filename=aa_stripe%2Futils.py - I am not really going to test admin.

Actually we could move read only admin into separate app so it would not pollute stats. But, IMO it would be art for art.

File: CHANGELOG.txt [hunk 1]
```diff
@@ -11,5 +11,19 @@ All notable changes to this project will be documented in this file.
 ### Added
 - related names to user model "stripe_tokens", "stripe_charges"
 - utils.py with get_latest_active_token_for_user
+- API to create StripeToken (Customer) out of stripe.js card authentication response. (uses Django-REST-Framework)
+### Changes
 - renamed StripeToken.content to StripeToken.stripe_js_response
-- added API to create StripeToken (Customer) out of stripe.js card authentication response. (uses Django-REST-Framework)
+
+## [0.2]
```
Comment (remik) [inline CHANGELOG.txt]: Please put newest on top. I missed it earlier. Sorry.

File: aa_stripe/api.py [hunk 2]
```diff
@@ -1,10 +1,50 @@
...
+
+class WebhookAPI(CreateAPIView):
+    queryset = StripeWebhook.objects.all()
+    serializer_class = StripeWebhookSerializer
+    permission_classes = (AllowAny,)
+
+    def post(self, request, *args, **kwargs):
+        payload = request.body.decode("utf-8")
+        sig_header = request.META.get("HTTP_STRIPE_SIGNATURE")
+        event = None
```
Comment (remik) [inline aa_stripe/api.py]: You don't need it.

File: aa_stripe/utils.py [hunk 3]
```diff
@@ -1,7 +1,40 @@
-from aa_stripe.models import StripeToken
+from aa_stripe.models import StripeCustomer
+from django.contrib import admin
 
 
-def get_latest_active_token_for_user(user):
-    """Returns last active stripe token for user"""
-    token = StripeToken.objects.filter(user_id=user.id, is_active=True).last()
-    return token
+def get_latest_active_customer_for_user(user):
```
Comment (remik) [inline aa_stripe/utils.py]: Move it to `StripeCustomer` as classmethod.

**Pull request merged**